core/modules/qdisp/QueryRequest.cc

```diff
@@ -58,7 +58,7 @@ inline void unprovisionSession(XrdSsiSession* session) {
 }
 
 ////////////////////////////////////////////////////////////////////////
-// QueryRequest::Canceller
+// QueryRequest::Canceler
 ////////////////////////////////////////////////////////////////////////
 class QueryRequest::Canceller : public util::VoidCallable<void> {
```

Comment:
Please, just make it consistent whatever you pick. Personally, I prefer double ell--maybe it reflects my roots in electrical engineering (never seen "echo cancelation" or "noise cancelation"--it's always "echo cancellation" and "noise cancellation"), but my days are numbered anyway.